app/templates/reading_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base_root.html" %}
+
+{% block navlinks %}
+  {{ super() }}
+{% endblock %}
+
+{% block main_body %}
+
+<div class="container">
+<div id="readinglist">
+    <div class="col-lg-10 ml-auto mr-auto">
+    	<h2 class="text-left title text-black">Your Book List</h2>
+        <div class="thumbnail">
+        	<div class ="row">
+        		<div class ="col-2">
+        			<div class="img-container">
+            			<img class="rounded float-left" src="https://images-na.ssl-images-amazon.com/images/I/51HTxer-rdL._SY346_.jpg" alt="" />
+            		</div>
+            	</div>
+	            <div class="col-10">
+	                <h4>The Hundred Year Flood</h4>
+	                <p>Matthew Salesses</p>
+	                <p>3.3 ★ ★ ★ ☆ ☆</p>
+                	<a class="btn btn-primary text-white" href="#">See Book Details</a>
```

Comment:
For `btn-primary` you don't need the `text-white` class because the text is white anyways ;-)

Reply:
same for all other occurences